### DISCUSSION TITLE
Readme.md for initialization

### DISCUSSION THREAD

**Discussion by OnlyArcoss** [adonisjs/core#4825] [Ideas]
There is no `Readme.md` after initializing a new project. I think it would be a good idea to include a default README file with useful links and commands (e.g., Discord links, documentation, start commands, etc.).

**Problem:** There is already a `Readme.md` file in the starter kits, but it gets deleted during initialization by the `create-adonis` script.

**Solutions:** Add an `init.md` file in the starter kits and rename it to `Readme.md` after initialization.

Here is an example of a README file below:

<p align="center">
<a href="https://adonisjs.com/" target="_blank"><img src="https://avatars.githubusercontent.com/u/13810373?s=200&v=4" width="120" alt="AdonisJS Logo" /></a>
</p>
<p align="center">
<a href="https://www.npmjs.com/package/@adonisjs/core" target="_blank"><img src="https://img.shields.io/npm/v/@adonisjs/core.svg" alt="NPM Version" /></a>
<a href="https://github.com/adonisjs/core" target="_blank"><img src="https://img.shields.io/github/license/adonisjs/core.svg" alt="License" /></a>
<a href="https://github.com/adonisjs/core" target="_blank"><img src="https://img.shields.io/github/stars/adonisjs/core.svg" alt="Stars" /></a>
<a href="https://discord.gg/vDcEjq6" target="_blank"><img src="https://img.shields.io/discord/423256550564691970.svg?logo=discord" alt="Join Discord" /></a>
<a href="https://docs.adonisjs.com" target="_blank"><img src="https://img.shields.io/badge/docs-Documentation-blue" alt="Documentation" /></a>
</p>

## Description
**AdonisJS** is a TypeScript-first web framework for building web apps and API servers. It comes with support for testing, modern tooling, an ecosystem of official

## Run and Build
```bash
# development mode
npm run dev

# production mode
npm start

# build for production
npm run build
```

## Run tests
```
npm run test
```

## Deployment

For detailed deployment instructions, refer to the [AdonisJS Deployment Guide](https://docs.adonisjs.com/guides/getting-started/deployment#deployment).

**Comment by thetutlage** [comment]
Yup, sounds like a good idea. Can you open PRs in the official starter kits for the same?

**Reply by OnlyArcoss** [reply]
Okay, I created the PR on the starter kit, but we shouldn't merge it until we've modified the initialization script to rename init.md to Readme.md.
